src/main/Member.js

```diff
@@ -798,11 +800,12 @@ export default class Member {
      * @param {string} accountId - id of the account
      * @param {string} offset - where to start looking
      * @param {int} limit - how many to retrieve
+     * @param {string} keyLevel - key level
      * @return {Promise} transactions - Transactions
      */
-    getTransactions(accountId, offset, limit) {
+    getTransactions(accountId, offset, limit, keyLevel) {
```

Comment:
can we make keyLevel optional? If it's equal to undefined, set to LOW or something.

Reply:
It's already the case, in keyLevel's setter it sets it to LOW if it's undefined.